environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
```

Comment:
How specific is this requirement?  Will the make in Apple's XCode Command Line Tools work?